src/node/rpc/member_frontend.h

```diff
@@ -1589,7 +1589,7 @@ namespace ccf
         };
       make_endpoint(
         "ack/update_state_digest", HTTP_POST, json_adapter(update_state_digest))
-        .set_auto_schema<void, StateDigest>()
+        .set_auto_schema<void, std::vector<uint8_t>>()
```

Comment:
I think we should make this a base64 string while we're updating it.

Reply:
Should we even make it a string all the way down, and have that _be_ the digest?

Reply:
That will certainly help for auditing too so I would say yes.

Reply:
We seem to use hex for digests pretty much everywhere else, so I'm going with that for now.